Resources/Prototypes/_NF/Entities/Objects/Tools/emag.yml

```
@@ -1,3 +1,19 @@
+- type: entity
+  parent: EmagUnlimited
+  id: EmagPass
+  name: cryptographic sequencer pass
```

```suggestion
  name: disposable cryptographic sequencer
```
I don't know if "pass" really implies disposable here.